templates/main.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>candidatos.info</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style>
+        .mainForm {
+            margin: auto;
+            width: 50%;
+            border: 3px solid grey;
+            padding: 10px;
+	}
+
+        .form-group {
+            margin: 10px;
+        }
+    </style>
+</head>
+
+<body>
+    <form id="selections" name="mainForm" class="mainForm" method="POST" enctype="multipart/form-data" action="/profiles">
+        <div class="form-group">
+            <label for="sel1">Estado</label>
+            <select name="stateForm" class="form-control" id="stateForm" class="stateForm">
+                <option>Selecione um estado</option>
+                {{ range .States }}
+		<option>{{ . }}</option>
+                {{ end }}
+            </select>
+        </div>
+        <div class="form-group">
+            <label for="sel1">Cidade</label>
+            <select name="cityForm" class="form-control" id="cityForm" class="cityForm">
+                <option>Selecione uma cidade</option>
+            </select>
+        </div>
+        <div class="form-group">
+            <label for="sel1">Role</label>
+            <select name="roleForm" class="form-control" id="rolesForm" class="rolesForm">
+                <option>Selecione um tipo de candidato</option>
+                <option>Vereador</option>
+                <option>Prefeito</option>
+            </select>
+        </div>
+    </form>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script>
+        var cities = {
```

Comment:
Acho que essa lógica vai mudar um pouco. Talvez já deixar engatilhado fazer a chamada e popular a opção das cidades depois que o estado for selecionado